Website/Packages/datax-pipeline/src/modules/flowDefinition/components/flowDefinitionPanel.jsx

```diff
@@ -318,7 +351,42 @@ class FlowDefinitionPanel extends React.Component {
         );
     }
 
+    renderScheduleTab() {
+        if (this.props.input.mode === Models.inputModeEnum.streaming){
+            return null;
+        }
+        else {
+            return (                               
+                <VerticalTabItem linkText="Schedule" itemCompleted={this.props.scheduleValidated}>
+                <ScheduleSettingsContent
+                    batchList={this.props.flow.batchList}
+                    selectedBatchIndex={this.props.selectedBatchIndex}
+                    onNewBatch={this.props.onNewBatch}
+                    onDeleteBatch={this.props.onDeleteBatch}
+                    onUpdateSelectedBatchIndex={this.props.onUpdateSelectedBatchIndex}
+                    addBatchButtonEnabled={this.state.addBatchButtonEnabled}
+                    deleteBatchButtonEnabled={this.state.deleteBatchButtonEnabled}
```

Comment:
nit: out of curiosity, I see in most of the files I've reviewed so far for the new batch support that you add newlines above and below your newly added members. Can you remove these new blank lines? Its not needed and if future folks were to keep doing this for every feature they add, it will just add a bunch of unnecessary blank lines sprinkled across our codebase.

Reply:
Prettier fixed this :)